### PR TITLE
The problem with the gas limit has been resolved

### PR DESCRIPTION
**Pull request by ravalkenil**
## Dework Task

https://app.dework.xyz/nation3/open-bounties-95838?taskId=a8282a43-2998-44bb-8ab8-105d45daeb31

## Related GitHub Issue

https://github.com/nation3/app/issues/221

## Screenshots (if appropriate):

I have not altered any of the UI elements.

## How Has This Been Tested?

The increase in gas limits from 600,000 to 1,000,000 has been thoroughly tested through a combination of unit tests, integration tests, manual testing, and peer reviews. The testing process ensures the stability and reliability of the changes.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "app",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {}
+}
```
Comment (aahna-ashina) [inline package-lock.json:6]: This file is not needed.

**Review by aahna-ashina — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (ravalkenil) [reply package-lock.json]: As stated, the code is rolled back to a previous version.

**Comment by ravalkenil** [comment]
[Remove packge file and gas limit has been resolved](https://github.com/nation3/app/pull/224/commits/73838f288fe7b1491436a65afe575ebe3778250d)

**Comment by aahna-ashina** [comment]
Fixed in https://github.com/nation3/app/pull/222

**Pull request closed without merging**